lib/wifi_user/gateway/gov_notify_sms.rb

```diff
@@ -0,0 +1,20 @@
+require 'notifications/client'
+
+class WifiUser::Gateway::GovNotifySMS
+  def initialize(api_key, template_id)
+    @client = Notifications::Client.new(api_key)
+    @template_id = template_id
```

Comment:
I'm torn between this being at initialisation or execution.

On the one hand, it allows a more static class structure if each SMS template we have as a counter part gateway:

```
send_credentials_sms = GovNotifySMS(api_key, config['notify_sms_template_ids']['credentials'])
```
but then this will result in needing to restructure the `sms_template_finder`.

Think by writing this, I've realised that there should be one gateway and multiple use-cases.